src/wasm/benchmark.rs

```diff
@@ -92,7 +92,7 @@ async fn blend_images_benchmark_js(
     let write_time = end - start_write;
 
     log_benchmark(
-        algorithm.unwrap_or("multiplicative".to_string()),
+        algorithm.unwrap_or_else(|| "multiplicative".to_string()),
```

Comment:
Not related to tests -> different PR. You can make the clippy stuff from all PRs in a single one.